### ISSUE TITLE
Path to smcutil hard coded

### ISSUE THREAD

**Issue by alexwhittemore** [godly-devotion/charge-limiter#1]
I saw your post over on the Al Dente issue and figured I'd give this a try, since I didn't understand exactly how you'd made it from your description. I tried to run it from my downloads folder and got this: 

<img width="532" alt="Screen Shot 2020-05-28 at 1 51 02 PM" src="https://user-images.githubusercontent.com/332827/83175714-31109200-a0d1-11ea-985b-c3f09cc63e72.png">

**Comment by godly-devotion** [comment]
Sorry about that, relative paths are a tricky thing. I've made some updates and released a new version (same version number though). Try that version and let me know if that works.

**Comment by alexwhittemore** [comment]
Sure looks like it works - I like it! 

I know I did a little funnybusiness on the Alfred workflow - I actually wrapped the terminal command call in applescript so that osascript would be responsible for privilege elevation - that way, it prompts you either for a password or for touch ID if you have it. I wonder if there's an easy tweak to avoid typing a password in this case if you don't have to? Anyway, totally keeping this around, thanks!

Random aside - it looks like, if you set the max limit to a value lower than your current state of charge, charging behavior won't change until you re-plug power. Like the PMIC only checks the SMC value on power-attached events.
